pkgs/applications/science/math/R/default.nix

```
@@ -33,6 +36,10 @@ stdenv.mkDerivation rec {
     substituteInPlace configure --replace "-install_name libR.dylib" "-install_name $out/lib/R/lib/libR.dylib"
   '';
 
+  # `null` only to not trigger mass rebuilds; on next change of this derivation,
+  # simply change to `dontDisableStatic = static`.
+  dontDisableStatic = if static then true else null;
```

R packages are tend to be cheap to compile, so I'd actually prefer straight-forward and readable source code here. The whole "don't disable" double negation here is bad enough as is. :-/

```suggestion
  dontDisableStatic = static;
```

---

OK, done.